sidebars.js

```diff
@@ -1,165 +1,191 @@
-                        'getting-started-acmecorp/bring-your-own-idp/deploy-peoplefinder-auth0',
-                    ],
-                },
-                'getting-started-acmecorp/install-onebox'
-            ]
+          type: 'category',
+          label: 'Explore ABAC Policies',
+          collapsed: true,
+          items: [
+            'getting-started-acmecorp/explore-abac-policies/overview',
+            'getting-started-acmecorp/explore-abac-policies/creating-an-abac-policy',
+            'getting-started-acmecorp/explore-abac-policies/update-app-and-review',
+          ],
         },
         {
-            Console: [
-                'aserto-console/introduction',
-                'aserto-console/connections',
-                'aserto-console/directory',
-                'aserto-console/policies',
-                'aserto-console/view-policies',
-                'aserto-console/manage-tenant',
-            ],
+          type: 'category',
+          label: 'Bring Your Own IDP (Auth0)',
+          collapsed: true,
+          items: [
+            'getting-started-acmecorp/bring-your-own-idp/bring-your-own-idp',
+            'getting-started-acmecorp/bring-your-own-idp/set-up-auth0',
+            'getting-started-acmecorp/bring-your-own-idp/extend-user-attributes',
+            'getting-started-acmecorp/bring-your-own-idp/deploy-peoplefinder-auth0',
+          ],
         },
+        'getting-started-acmecorp/install-onebox'
+      ]
+    },
+    {
+      Console: [
+        'aserto-console/introduction',
+        'aserto-console/connections',
+        'aserto-console/directory',
+        'aserto-console/policies',
+        'aserto-console/view-policies',
+        'aserto-console/manage-tenant',
+      ],
+    },
+    {
+      'Authorizer Guide': [
+        'authorizer-guide/overview',
+        'authorizer-guide/authz',
+        'authorizer-guide/is',
+        'authorizer-guide/identity-context',
+        'authorizer-guide/policy-context',
+        'authorizer-guide/resource-context',
+        'authorizer-guide/query',
+        'authorizer-guide/display-state-map',
+        'authorizer-guide/decisiontree',
+        'authorizer-guide/dir',
+        'authorizer-guide/policy',
+      ],
+    },
+    {
+      'Quickstarts': [
         {
-            'Authorizer Guide': [
-                'authorizer-guide/overview',
-                'authorizer-guide/authz',
-                'authorizer-guide/is',
-                'authorizer-guide/identity-context',
-                'authorizer-guide/policy-context',
-                'authorizer-guide/resource-context',
-                'authorizer-guide/query',
-                'authorizer-guide/display-state-map',
-                'authorizer-guide/decisiontree',
-                'authorizer-guide/dir',
-                'authorizer-guide/policy',
-                'authorizer-guide/built-ins',
-            ],
+          type: 'category',
+          label: 'Javascript',
```

Comment:
I don't think we should have a "Javascript" category here - it's too deep of a hierarchy considering that you only have a single Quickstart at the moment. I would just have "React and Node" be right under "Quickstarts"